tests/integration/test_storage_s3/test.py

```diff
@@ -1776,7 +1776,7 @@ def test(engine, setting):
     assert int(res) == 0
 
     res = instance.query(
-        f"select * from url('http://{started_cluster.minio_host}:{started_cluster.minio_port}/{bucket}/skip_empty_files{{11|1|22}}.parquet') settings engine_url_skip_empty_files=1"
+        f"select * from url('http://{started_cluster.minio_host}:{started_cluster.minio_port}/{bucket}/skip_empty_files{{11|1|22}}.parquet', auto, 'number UInt64') settings engine_url_skip_empty_files=1"
```

Comment:
Two files don't exists, third file is empty. There is no way to infer the schema. It worked only because of schema inference cache filled by previous queries for file `skip_empty_files1.parquet`. Sometimes for some reason we cannot get schema from cache (maybe because of parallel tests running, not sure) and query fails, better to specify structure manually